bot/code_review_bot/revisions.py

```diff
@@ -406,29 +405,27 @@ def is_decision_task(task):
         decision_task = next(filter(is_decision_task, tasks.values()), None)
         assert decision_task is not None, "Missing decision task"
 
-        # Use mercurial infos for local revision
+        # Match the supported repositories from settings
+        # with the decision task environment to get the mercurial information
         decision_env = decision_task["task"]["payload"]["env"]
-        if "GECKO_HEAD_REPOSITORY" in decision_env:
-            # Mozilla-Central Try
-            self.mercurial_revision = decision_env.get("GECKO_HEAD_REV")
-            self.repository = decision_env["GECKO_HEAD_REPOSITORY"]
-            # mozilla-unified is used in the Decision task payload
-            # but that is not the "real" target repository
-            self.target_repository = REPO_MOZILLA_CENTRAL
-
-        elif "NSS_HEAD_REPOSITORY" in decision_env:
-            # NSS Try
-            self.mercurial_revision = decision_env.get("NSS_HEAD_REVISION")
-            self.repository = decision_env["NSS_HEAD_REPOSITORY"]
-            # Unfortunately the NSS decision task does not expose the target repository
-            self.target_repository = REPO_NSS
-
-        else:
-            raise Exception("Unsupported decision task")
-
-        # Save mercurial revision
-        assert self.mercurial_revision is not None, "Missing try revision"
-        logger.info("Using Try mercurial revision", rev=self.mercurial_revision)
+        for repository in settings.repositories:
```

Comment:
What if we used the information in the Phabricator revision to detect which repository to use?

Reply:
It won't work because Phabricator has no knowledge of the try repositories, and i need the try & target repos + mercurial revision.

Reply:
I meant just in order to detect which try repository to use, instead of relying on the presence of the env variable in the decision task.

Reply:
Done: i'm using the Phabricator repo info to match the decision task